.github/ISSUE_TEMPLATE/faucet.yml

```diff
@@ -0,0 +1,42 @@
+name: "Issue with the Faucet"
+description: Create a new ticket to report an issue with the faucet
+title: "📡 Problem with the Faucet"
+labels: ["client"]
+body:
+  - type: dropdown
+    id: faucet
+    attributes:
+      label: Faucet Type
+      description: What faucet where you using?
+      options:
+        - Web
+        - Matrix
+    validations:
+      required: true
+  - type: textarea
+    id: summary
+    attributes:
+      label: "Problem"
+      description: Please provide a clear and concise description of the problem.
+    validations:
+      required: true
+  - type: input
+    id: address
+    attributes:
+      label: Address
+      description: What was the address where you encountered the problem?
+      placeholder: ex. 5CPkfj09f4WWF4320jjFcFdmCF1CSLbhDeg3gbA
```

Comment:
I added the address in the case that we need to search if the problem is related to the address itself (has enough tokens or is blacklisted).

This won't be a security issue, right?

Reply:
My only suggestion is to explicitly mark it as optional.
It's not configured to be required already, but I would make it more apparent, e.g.:

`(optional) What was the address where you encountered the problem?`